examples/set-namespace-simple/README.md

```diff
@@ -37,11 +36,9 @@ key-value pair:
 Invoke the function by running the following commands:
 
 ```shell
-$ kpt fn render set-namespace-simple
+$ kpt fn render set-namespace
 ```
 
 ### Expected result
 
-Check all resources have `metadata.namespace` set to `example-ns`:
-
-[`set-namespace`]: https://catalog.kpt.dev/set-namespace/v0.1/
```

Comment:
If we remove this, the link in line 5 will be broken.
We can use `https://catalog.kpt.dev/set-namespace/v0.3/`, if we plan to release it very soon.

Reply:
Fixed.